### ISSUE TITLE
Cannot create compressed css without workaround configuration in build.yaml

### ISSUE THREAD

**Issue by denniskaselow** [dart-league/sass_builder#66]
When building a release version of my project using sass_builder 2.1.4 on

`Dart SDK version: 2.19.0-201.0.dev (dev) (Wed Sep 14 14:58:07 2022 -0700) on "windows_x64"`

I get the following warning:

> [WARNING] sass_builder on web/styles.scss:
Unknown outputStyle provided: "compressed". Supported values are: "expanded" and "compressed". The default value of "OutputStyle.expanded" will be used.

This happens because these checks will always be `false`:

https://github.com/dart-league/sass_builder/blob/d24cc29c6a34987cc44622e2d6492246034d1ed3/lib/sass_builder.dart#L62-L67

because `sass.OutputStyle.compressed.toString()` resolves to the string `OutputStyle.compressed` and is compared to `compressed`.

I'm pretty sure it did work at some point in the past. (maybe `toString()`/`name` changed in the enhanced enums release?)

The workaround I'm using is to set `outputStyle` in `build.yaml` to the actual string it's compared to instead of relying on default values: 

```yaml
targets:
  $default:
    builders:
      sass_builder:
        options:
          outputStyle: OutputStyle.compressed
```

**Comment by simolus3** [comment]
This is fixed in version `2.1.5`! You'll need to revert your `build.yaml` workaround with that version though, only the "short" form of `compressed` is supported.

**Comment by denniskaselow** [comment]
Whoops, this fix introduced a different bug. The default value in development mode is now in the wrong format (release mode and values in `build.yaml` work correctly).

The line https://github.com/dart-league/sass_builder/blob/eee58628615542fc86ad0a74270b3e89229f5188/lib/sass_builder.dart#L36 now needs to be `_outputStyle = outputStyle ?? _defaultOutputStyle.name,` otherwise it won't be found in `_outputStylesByName`.

EDIT: Though the end result is the same, an expanded css. It's just the warning that's unnecessary:

> Unknown outputStyle provided: "OutputStyle.expanded". Supported values are: "expanded" and "compressed". The default value of "OutputStyle.expanded" will be used.

The warning message should use `name` too, for 'the default value of ... will be used'.

**Comment by simolus3** [comment]
Oh dang, good catch! I've fixed this in 0704e1c15d003c3195034c02bf609c4fa54f6e2a (version 2.2.1).
